packages/ember-htmlbars/tests/helpers/with_test.js

```diff
@@ -1,262 +1,13 @@
 import Ember from 'ember-metal/core';
 import EmberView from 'ember-views/views/view';
-import run from 'ember-metal/run_loop';
-import EmberObject from 'ember-runtime/system/object';
-import { set } from 'ember-metal/property_set';
 import compile from 'ember-template-compiler/system/compile';
 import { runAppend, runDestroy } from 'ember-runtime/tests/utils';
 
 var view, lookup;
 var originalLookup = Ember.lookup;
 
-function testWithAs(moduleName, templateString, deprecated) {
```

Comment:
Note for self: `templateString` is always `{{#with person as |tom|}}{{title}}: {{tom.name}}{{/with}}`